src/signTx.c

```diff
@@ -975,6 +1050,14 @@ static void signTx_handleCertificate_ui_runStep()
 			        this_fn
 			);
 			break;
+		case STAKE_CREDENTIAL_KEY_HASH:
+			ui_displayHexBufferScreen(
```

Comment:
We should use bech32 with "stake_vkh" prefix.

Reply:
This is being changed in Trezor, too. Perhaps make a list of UI changes and consult with David that we display everything consistently with Trezor.

Reply:
I'm collecting all of them and will do them all after comparing.